language-examples/csharp/rest/RestApiKeyQueries.cs

```diff
@@ -72,16 +83,37 @@ private static void Query(long customerId, long corpusId, String query, String s
                     request.Headers.Add("customer-id", customerId.ToString());
 
                     HttpResponseMessage response = client.Send(request);
-                    String result = response.Content.ReadAsStringAsync().Result;
+                    string result = response.Content.ReadAsStringAsync().Result;
+                    JObject resultObj = JObject.Parse(result);
+                    JToken? responseSetArray = resultObj["responseSet"];
+                    if (responseSetArray == null)
+                    {
+                        throw new Exception("No results found");
+                    }
+                    foreach (var responseSet in responseSetArray)
```

Comment:
Don't you need to check the status object here too inside responseset like you did in the grpc example?

Reply:
Done.